validation/package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JSON is not sorted](https://app.codacy.com/manual/jrtapsell/MetabaseAutoCovid/pullRequest?prid=5322169)